rollup.base.js

```diff
@@ -0,0 +1,17 @@
+import commonjs from 'rollup-plugin-commonjs';
+import resolve from 'rollup-plugin-node-resolve';
+
+export default {
+    input: './node_modules/@mparticle/web-kit-wrapper/index.js',
+    output: {
+        format: 'umd',
+        exports: 'named',
+        strict: false,
```

Comment:
let's make this true per our discussion

Reply:
let's get this change done first thing in the morning, then we'll merge, and then can update optimizely, doubleclick, and GTM to use rc7.  build those, and see if tests continue to pass